### ISSUE TITLE
multiple variable search

### ISSUE THREAD

**Issue by tadpohl1972** [Sam-Martin/servicenow-powershell#20]
Is it possible to search two attributes at the same time.

aka Get-ServiceNowIncident -MatchContains @{assignment_group='Corporate' -and location='texas' -and active='true'}

I would like to get specific tickets using multiple hashtables

**Comment by Rick-2CA** [comment]
Yes, but `MatchContains` expects a hash table so the format would be `@{assignment_group='Corporate';location='Texas';Active='True'}`.
